README.md

```diff
@@ -1,2 +1,13 @@
 # privacy-security
-Privacy &amp; Security Principles, Documents and Testing
+
+This repository is for maintaining...
```

Comment:
We believe that transparency in our software is an important part of ensuring security and privacy. This repository is meant to provide a public place for OSS developers, interested security and privacy experts to go to understand Path Check's: